src/services/doppler-system-usage-api-client.double.ts

```diff
@@ -0,0 +1,21 @@
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+import { timeout } from '../utils';
+import { SystemUsage } from './doppler-system-usage-api-client';
+
+export interface DopplerSystemUsageApiClient {
+  getUser(): Promise<ResultWithoutExpectedErrors<SystemUsage>>;
```

Comment:
What do you think about it?

```suggestion
  getUserSystemUsage(): Promise<ResultWithoutExpectedErrors<SystemUsage>>;
```